signer/core/apitypes/types.go

```diff
@@ -117,15 +139,34 @@ func (args *SendTxArgs) ToTransaction() *types.Transaction {
 		to = &dstAddr
 	}
 
-	var input []byte
-	if args.Input != nil {
-		input = *args.Input
-	} else if args.Data != nil {
-		input = *args.Data
-	}
-
 	var data types.TxData
 	switch {
+	case args.BlobHashes != nil:
+		al := types.AccessList{}
```

Comment:
This will always add an empty access list? Could be simpler by assigning to `data.AccessList` only if `args.AccessList` non-nil.

Reply:
Ahh, wait, so the issue is actually that `args.AccessList` is defined as pointer. That's not necessary because `AccessList` is a slice type, which already has a `nil` state.

Reply:
I think I just copy-pasted this from the `TransactionArgs`

Reply:
Yeah, so maybe it's wrong there too.